package.json

```
@@ -124,7 +124,7 @@
     "generate_scheme": "./tasks/generate_scheme.js"
   },
   "peerDependencies": {
-    "@vkontakte/icons": "^1.184.0",
+    "@vkontakte/icons": "^1.208.0 || ^2.0.2",
```

А зачем поднимать v1?

```suggestion
    "@vkontakte/icons": "^1.184.0 || ^2.0.1",
```

---

И правда, это я опять переусердствовала, спасибо за бдительность. 😉